### ISSUE TITLE
v2.6 UI Tech Preview Requirements

### ISSUE THREAD

**Issue by deniseschannon** [rancher/dashboard#3579]
- Add ability to indicate the RKE2 is a a tech preview
- Add ability to indicate Cluster Templates is a tech preview
- Hide Cluster Templates from UI when rke2 feature flag is disabled
- When launching Rancher for the first time, default to RKE1 when RKE2 feature flag is enabled

**Comment by slickwarren** [comment]
notes for QA: 
* RKE2 templates are added via git repos now. You can use the [default example from Strongmonkey](https://github.com/StrongMonkey/cluster-template-examples) or use the one[ I've fork'd from that for QA purposes](https://github.com/slickwarren/cluster-template-examples/tree/v2charts-template) and use the `v2charts-template` branch (currently, there are a few errors in both repos. So if testing, stick with Digital Ocean, single node.

**Comment by davidnuzik** [comment]
**Reproduction:**
Not required.

**Validation Successful:**
Rancher version: master-head `30574c2a9`
Rancher cluster type: single-node rancher server in docker container
Docker version: 19.03

Validation checks:

> Add ability to indicate the RKE2 is a a tech preview

If you switch the RKE1 <-> RKE2/K3s toggle on the Cluster Creation page to RKE2/K3s, then provisioning is mentioned as Tech Preview as pictured:
![image](https://user-images.githubusercontent.com/45179589/128089893-c5b9b94c-74cd-4d5b-9b7a-c5dde6c2de72.png)
This affects Amazon EC2, Azure, DigitalOcean, Linode, vSphere, and Custom clusters. It does NOT affect hosted k8s providers.
I also 

> Add ability to indicate Cluster Templates is a tech preview

Already tested by Caleb - I got the go ahead from Sowmya to skip this.

> Hide Cluster Templates from UI when rke2 feature flag is disabled

Already tested by Caleb - I got the go ahead from Sowmya to skip this.

> When launching Rancher for the first time, default to RKE1 when RKE2 feature flag is enabled

Already tested by Sowmya - I got the go ahead from Sowmya to skip this.
